StarRailBattleSim/src/relics/ornament/TheWondrousBananAmusementPark.java

```diff
@@ -0,0 +1,21 @@
+package relics.ornament;
+
+import characters.AbstractCharacter;
+import relics.AbstractRelicSetBonus;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+/**
+ * Placeholder, don't use. Not sure how to check if the wearer has a summon.
```

Comment:
will most likely need to add a specific boolean flag in AbstractCharacter called `HasSummon` and manually set it for characters with summons